packages/vite/src/executors/dev/executor.ts

```diff
@@ -42,7 +42,7 @@ export default async function runExecutor(
         ? false
         : options.configFile
         ? joinPathFragments(`${context.root}/${options.configFile}`)
-        : false,
+        : undefined,
```

Comment:
@DominikPieper  this is the dev issue. Oversight on the change

Reply:
Okay great, thanks :-)

Reply:
Can you just check the migration. Never created a migration script for NX before

Reply:
Looks good to me. The most verbose stuff for migrations mostly is checking it's a project from the right plugin ^^'